mlflow/recipes/utils/wrapped_recipe_model.py

```diff
@@ -0,0 +1,35 @@
+import mlflow
+from mlflow.pyfunc import PythonModel
+import pandas as pd
+import numpy as np
+
+
+class WrappedRecipeModel(PythonModel):
```

Comment:
Is this model only meant for classification? Its name suggests it's universal. Can we rename it to `RecipeClassificationModel` or something?

Reply:
Shouldn't a base class + subclass approach work better? Almost all contexts in this class suggest it's meant for a classifier model. I don't see it can be easily extended for "more options" in the future.

Reply:
I think once we have a broader usecase, we can design for it better. Hopefully we won't need all of this once we have microsoft proposal merged as well to add additional functions to pyfunc model